docs/guides/add-new-provider.md

```diff
@@ -221,6 +222,41 @@ _e.g. `https://api.example.com/?accessKey=<apikey>`_
 - `in` must be set to `query`
 - `name` is the query param name that holds the API key (e.g. `accessKey`)
 
+### Digest Auth {#digest-auth}
+
+_e.g. `Authorization: Digest <credentials>` in headers_
+
+Use the following scheme with an arbitrary ID which can be referenced later from the mapping.
+
+```json title="<provider-name>.provider.json" {10-16}
+{
+  "name": "<provider-name>",
+  "services": [
+    {
+      "id": "api",
+      "baseUrl": "https://api.example.com"
+    }
+  ],
+  "defaultService": "api",
+  "securitySchemes": [
+    {
+      "id": "<scheme-id>",
+      "type": "http",
+      "scheme": "digest"
+    }
+  ]
+}
+```
+
+_Replace the security scheme `id` value in the example with your own ID._
+
+#### Optional parameters:
+- `statusCode` HTTP status code that should be returned from initial call (containing challenge header). Default value is `401`.
+- `challengeHeader` Case insensitive name of header containing challenge from the server. Default value is `www-authenticate`.
+- `authorizationHeader` Case insensitive name of header containing authorization. Default value is `Authorization`.
```

Comment:
I would remove this section and added a link to the relevant reference section instead. I suppose these parameters will be rarely needed.